dataset.ttl

```diff
@@ -1224,188 +1292,437 @@ data:YarramanSubprovince
             geox:hasRole grole:boundary ] ;
     time:hasTime ischart:Carboniferous,
-        ischart:UpperDevonian ;
+        ischart:Devonian ;
     sdo:name "Yarraman Subprovince"@en .    
+   <<< in here twice; comapre both entries #

```

Comment:
Delete the duplicate entry please John, whichever is the lesser or has the least info, or the second copy if they are exactly the same